src/test/java/com/cloudant/tests/DatabaseTest.java

```diff
@@ -71,6 +74,16 @@ public void permissions() {
 
     }
 
+    /**
+     * Test that when called against a DB that is not a Cloudant service
+     * an UnsupportedOperationException is thrown
+     */
+    @Test(expected = UnsupportedOperationException.class)
+    @Category({RequiresCouch.class, RequiresCloudantLocal.class})

```

Comment:
Is it possible to make the whole class `@Category(RequiresDB.class)` and then only add in the necessary others? My guess is that the categories compose in this way but I'm not sure.